code/modules/mob/living/carbon/xenomorph/facehuggers.dm

```diff
@@ -411,7 +490,7 @@
 		user.disable_lights(sparks = TRUE, silent = TRUE)
 		var/stamina_dmg = user.maxHealth * 2 + user.max_stamina_buffer
 		user.apply_damage(stamina_dmg, STAMINA) // complete winds the target
```

Comment:
This is fucking awfull please change, this makes huggers able to instakill people due to your changes to stamina